modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.js

```diff
@@ -533,25 +537,14 @@ const FrontendDataSet = ({
 			setHighlightedItemsValue([]);
 		}
 
-		if (
-			(nestedItemsReferenceKey && !nestedItemsKey) ||
-			(!nestedItemsReferenceKey && nestedItemsKey)
-		) {
-			logError(
-				'"nestedItemsKey" and "nestedItemsReferenceKey" params are both mandatory to manage nested items'
-			);
-		}
```

Comment:
Cleanup of unnecessary console logging, that adds dependencies and complicated code.